release-notes.txt

```diff
@@ -1,3 +1,622 @@
+RIOT-2021.01 - Release Notes
+============================
+RIOT is a multi-threading operating system which enables soft real-time
+capabilities and comes with support for a range of devices that are typically
+found in the Internet of Things: 8-bit and 16-bit microcontrollers as well as
+light-weight 32-bit processors.
+
+RIOT is based on the following design principles: energy-efficiency, soft
+real-time capabilities, small memory footprint, modularity, and uniform API
+access, independent of the underlying hardware (with partial POSIX compliance).
+
+RIOT is developed by an international open-source community which is
+independent of specific vendors (e.g. similarly to the Linux community) and is
+licensed with a non-viral copyleft license (LGPLv2.1), which allows indirect
+business models around the free open-source software platform provided by RIOT.
+
+
+About this release:
+===================
```

Comment:
On community aspects, is it worth noting, that during this release cycle we started migrating from a mailing list based communication to forum-based communication?

Reply:
yes, I think it's worth it

Reply:
done! 5696b545a96df1a7bee9ef11ad8eca9713a16ecd

Reply:
done! b11948ec0a3af96cf97ec341b892b725533593d6

I also added the forum next to IRC and Matrix